src/plugins/discover/public/application/main/utils/get_state_defaults.ts

```diff
@@ -48,7 +48,7 @@ export function getStateDefaults({
   const query = searchSource.getField('query') || data.query.queryString.getDefaultQuery();
   const sort = getSortArray(savedSearch.sort ?? [], indexPattern!);
   const columns = getDefaultColumns(savedSearch, config);
-  const chartHidden = Boolean(storage.get(CHART_HIDDEN_KEY));
+  const chartHidden: boolean | unknown = storage.get(CHART_HIDDEN_KEY);
```

Comment:
about `unknown` ... the only state except `true` | `false` could be `undefined`?

Reply:
we could manually change `hideChart` state to `string`, `object`, whatever.

Reply:
ah, yes, right 👍